org.gnome.powimod.aion-task.yml

```diff
@@ -0,0 +1,22 @@
+app-id: org.gnome.powimod.aion-task
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+
+command: aion-task
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--filesystem=xdg-data/aion-task/:create'
```

Comment:
it shouldn't need this at all.
arbitrary `xdg-data` is in general not allowed.

Reply:
This is the build failure:
```
        "finish-args-unnecessary-xdg-data-access"
```

But again I don't think you need it. If you do it's a bug, gtk is supposed to give you the right directories.